spk/rabbitmq/Makefile

```diff
@@ -32,7 +32,7 @@ include ../../mk/spksrc.spk.mk
 
 .PHONY: rabbitmq_extra_install
 rabbitmq_extra_install:
-	$(RUN) patch -i $(WORK_DIR)/../poststrip_patches/001_rabbitmq-server.patch $(STAGING_DIR)/lib/rabbitmq_server-$(SPK_VERS)/sbin/rabbitmq-server
+	# Create required directories
 	install -m 755 -d ${STAGING_DIR}/etc/
```

Comment:
this line is obsolete, as `install -d` is like `mkdir -p` that creates all the subdirectories.

and you can even create a bunch of directories with one call like:

`install -m 755 -d ${STAGING_DIR}/etc/rabbitmq	${STAGING_DIR}/etc/var`

But my question is: why not use 
`${STAGING_DIR}/etc` instead of `${STAGING_DIR}/etc/rabbitmq`
and
`${STAGING_DIR}/var` instead of `${STAGING_DIR}/etc/var`

Reply:
@DigitalBox98 I have reworked this myself.
I kept `${STAGING_DIR}/etc/rabbitmq` for `rabbitmq-env.conf` as it failed otherwise, but moved other config files to `${STAGING_DIR}/var` to benefit from the backup/restore on package updates implemented in the generich installer.